devicon.json

```
@@ -888,6 +888,26 @@
         "color": "#A80030",
         "aliases": []
     },
+    {
+        "name": "denojs",
+        "tags": [
+            "javascript",
+            "rust",
+            "language"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "original-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
```

The icon has one color already.
```suggestion
                "original",
                "original-wordmark"
```